src/tm2/tm-info.cpp

```diff
@@ -10,7 +10,9 @@
 
 #include "tm-info.h"
 #include "tm-device.h"
-#include "libusb.h"
+#ifndef __ANDROID__
+    #include "libusb.h"    
+#endif
```

Comment:
Unrelatedly I created a patch , e4e6927,  to remove this dependency a few hours ago as part of #6083.